tasks/zadanie_10.php

```diff
@@ -18,4 +18,14 @@
 
 $discountAmount = 0;
 
+if ($productsCost > 100) {
+    if ($type === "gold") {
+        $discountAmount = $productsCost * 0.2;
+    } elseif ($type === "silver") {
+        $discountAmount = $productsCost * 0.1;
+    } else {
+        $discountAmount = 0;
+    }
+} else "Wydałeś za mało!";
```

Comment:
```suggestion
}
```
Ogólnie w tym else to 
1: brakuje klamry :D 
2: przed tym "Wydałeś za mało!"; nie ma echo ani nic z tych rzeczy więc ten string w sumie to tak leci sobie w przestrzeń :D 

Ale też w sumie z tego drugiego powodu - zadanie pokazuje się zielone bo się ten komunikat nie wyświetla na ekranie ;)

Reply:
Faktycznie, błędnie ale dobrze:D